specification/web/resource-manager/Microsoft.DomainRegistration/2015-04-01/Domains.json

```diff
@@ -574,6 +574,51 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/renew": {
+      "post": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Renew a domain.",
+        "description": "Renew a domain.",
+        "operationId": "Domains_Renew",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of the domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Domain renewal was successful."
+          },
+          "202": {
+            "description": "Domain renewal in progress."
+          },
+          "204": {
+            "description": "Domain does not exist."
+          },
+          "400": {
```

Comment:
I'm not sure what you mean. Do you mean just changing the description from something like "Malformed domain renewal request." to "'400' - 'Bad request'." ?

Reply:
If you list a response code like this it will be treated as a valid response code. Thus you must not list error response codes but put all of them in meaningful descriptions in status code "default". E.g.: https://github.com/Azure/azure-rest-api-specs/blob/9ae8eeaa674385e8b387aceabd79eb00a5279d21/specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json#L63